ft/onto/race_mutli_choice_qa_ontology.py

```diff
@@ -0,0 +1,130 @@
+# ***automatically_generated***
+# flake8: noqa
+# mypy: ignore-errors
+# pylint: skip-file
+"""
+
+
+Automatically generated ontology race_mutli_choice_qa_ontology. Do not change manually.
+"""
+
+from forte.data.data_pack import DataPack
+from forte.data.ontology.top import Annotation
+from ft.onto.base_ontology import Document
+from typing import List
+from typing import Optional
+
+
+__all__ = [
+    "Passage",
+    "Option",
+    "Question",
+]
+
+
+class Passage(Document):
+    """
+
+    Attributes:
+        _passage_id (Optional[str])
+
+    """
+
+    def __init__(self, pack: DataPack, begin: int, end: int):
+        super().__init__(pack, begin, end)
+        self._passage_id: Optional[str] = None
+
+    def __getstate__(self): 
+        state = super().__getstate__()
+        state['passage_id'] = self._passage_id
+        return state
+
+    def __setstate__(self, state): 
+        state = super().__setstate__(state)
+        self._passage_id = state.get('passage_id', None) 
+
+    @property
+    def passage_id(self):
+        return self._passage_id
+
+    @passage_id.setter
+    def passage_id(self, passage_id: Optional[str]):
+        self.set_fields(_passage_id=passage_id)
+
+
+class Option(Annotation):
+    """
+
+
+    """
+
+    def __init__(self, pack: DataPack, begin: int, end: int):
+        super().__init__(pack, begin, end)
+
+    def __getstate__(self): 
+        state = super().__getstate__()
+        return state
+
+    def __setstate__(self, state): 
+        state = super().__setstate__(state)
+
+
+class Question(Annotation):
+    """
+
+    Attributes:
+        _options (Optional[List[int]])
+        _answers (Optional[List[int]])
+
+    """
+
+    def __init__(self, pack: DataPack, begin: int, end: int):
+        super().__init__(pack, begin, end)
+        self._options: Optional[List[int]] = []
+        self._answers: Optional[List[int]] = []
+
+    def __getstate__(self): 
```

Comment:
It looks like we don't need to do special get_state and set_state, since the Forte types are now all int.